hera_pspec/utils.py

```diff
@@ -505,3 +530,123 @@ def flatten(nested_list):
     """
     return [item for sublist in nested_list for item in sublist]
 
+
+def config_pspec_blpairs(uv_templates, pol_pairs, group_pairs, exclude_auto_bls=True, 
+                         exclude_permutations=True, bl_len_range=(0, 1e10), 
+                         bl_deg_range=(0, 180), xants=None, verbose=True):
+    """
+    Given a list of miriad file templates and selections for
+    polarization and group labels, construct a master list of
+    blpair-group-pol pairs using utils.construct_reds().
+
+    A baseline-pair is formed by self-matching unique-files in the
+    glob-parsed master list, and then string-formatting-in appropriate 
+    pol and group selections given pol_pairs and group_pairs. Those two
+    files are then passed to utils.calc_reds(..., **kwargs) to construct
+    the baseline-pairs for that particular file-matching.
+
+    Parameters
+    ----------
+    uv_templates : list
+        List of glob-parseable string templates, each of which must have
+        a {pol} and {group} field.
+
+    pol_pairs : list
+        List of len-2 polarization tuples to use in forming cross spectra.
+        Ex: [('xx', 'xx'), ('yy', 'yy'), ...]
+
+    group_pairs : list
```

Comment:
Not quite sure what a 'group' is in this context?

Reply:
Sure, I'll add some more text in the doc-string with examples